components/reports/precipitation/ReportPrecipChart.vue

```diff
@@ -82,11 +87,31 @@ export default {
 			}
 
 			let scatterTraces = {}
+
 			let symbols = {
 				'5modelAvg': 'circle',
 				'MRI-CGCM3': 'square',
 				'CCSM4': 'diamond',
 			}
+
+			let colors = {
+				'5modelAvg': {
+					'rcp45': 'rgb(190, 30, 30)',
+					'rcp60': 'rgb(210, 90, 90)',
+					'rcp85': 'rgb(230, 150, 150)',
+				},
+				'MRI-CGCM3': {
+					'rcp45': 'rgb(30, 30, 190)',
+					'rcp60': 'rgb(90, 90, 210)',
+					'rcp85': 'rgb(150, 150, 230)',
+				},
+				'CCSM4': {
+					'rcp45': 'rgb(140, 140, 30)',
+					'rcp60': 'rgb(180, 180, 90)',
+					'rcp85': 'rgb(210, 210, 150)',
+				},
```

Comment:
I really like this color choice and how it works -- you can see "model clusters" at a glance more easily, supported by the different shapes.  I dig it!

Reply:
But!  Can I suggest that you reverse RCP 4.5 / 8.5?  My eye sees the lighter pink as "cooler" than the brighter-red, but the RCP 8.5 model generally implies increased warming.  So "palest" = 4.5, "medium" = 6.0, "most saturated" = 8.5  (Do the same for PR please!!)

Reply:
Done in 0ab958a.